scss/_base_button.scss

```diff
@@ -17,9 +17,9 @@
     display: inline-block;
     font-size: 1rem;
     font-weight: 300;
-    line-height: map-get($line-heights, default-text);
+    line-height: 1;
     margin-bottom: $spv-inter--scaleable + 2 * $spv-nudge-compensation;
-    padding: $spv-nudge - $px $sph-intra;
```

Comment:
Won't removing this nudge break the baseline grid if the scaling changes

Reply:
Margin should to being the alignment, not padding. The scaleable adjustment is made from the `$spv-nudge-compensation` above.

Reply:
Cool, makes sense